### PR TITLE
Added Profile

### PR DESCRIPTION
**Pull request by wakenight123**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -3414,6 +3414,17 @@ a:visited {
  background-image: url("https://learn-verified.s3.amazonaws.com/david-elden-cover.jpg");
 }
 
+<<<<<<< HEAD
```
Comment (bhollan) [inline css/styles.css]: Hi!  Lines like this actually break the CSS.  They are a result of `git` having what's called a "merge conflict".  You should be ok if you delete this line, and it's mate below.

File: css/styles.css [hunk 2]
```diff
@@ -3414,6 +3414,17 @@ a:visited {
...
+#benny-hung-card {
+  background-image: url(http://i.imgur.com/T35VBUk.jpg?1)
+}
+
+#benny-hung-cover {
+  background-image: url(http://i.imgur.com/7xxwtz0.jpg?1)
+}
+
+=======
+>>>>>>> dc9bff577c68ab59456b6ad65d1703429e19948e
```
Comment (bhollan) [inline css/styles.css]: These two lines bookend the conflict areas resulting from a merge conflict.
Delete these two lines here as well as the mate above, and this should be ready to merge in!

File: css/styles.css [hunk 3]
```diff
@@ -3414,6 +3414,17 @@ a:visited {
  background-image: url("https://learn-verified.s3.amazonaws.com/david-elden-cover.jpg");
 }
 
+<<<<<<< HEAD
+#benny-hung-card {
+  background-image: url(http://i.imgur.com/T35VBUk.jpg?1)
```
Comment (bhollan) [inline css/styles.css]: The URL here is great!  It works and is fully functional.  However, for consistency's sake, it should probably be in `"`s.  Can you change that?

File: css/styles.css [hunk 4]
```diff
@@ -3414,6 +3414,17 @@ a:visited {
...
+}
+
+#benny-hung-cover {
+  background-image: url(http://i.imgur.com/7xxwtz0.jpg?1)
```
Comment (bhollan) [inline css/styles.css]: Same thing here with the `"`s

**Comment by bhollan** [comment]
Hi Benny!  Just looking through these pull requests to merge in and it looks like you have a couple of issues to resolve.  I've made notes here on github's file viewer.  If you need help, let us know!

CC: @aturkewi

File: css/styles.css (hunk 1 above)
Comment (wakenight123) [inline css/styles.css]: Hi Brian,

Thank you for informing me. I have deleted the extraneous lines, and added
"s, and I have committed these changes.

Best,
Benny

**Pull request merged**